src/components/base/components/DatePicker/CustomRangePicker.js

```diff
@@ -7,10 +8,11 @@ import { MM_DD_YYYY } from 'utils/constants/labels';
 
 import '../../less/datePicker.less';
 
-function RangePicker({ className, dateRange, onChange, ...props }) {
-  const { RangePicker: AntRangePicker } = DatePicker;
+function CustomRangePicker({ className, dateRange, onChange, ...props }) {
+  const { RangePicker } = DatePicker;
+
   return (
-    <AntRangePicker
+    <RangePicker
       allowClear={false}

```

Comment:
why we change this AntRangePicker to RangePicker , because we get better idea from AntRangePicker , so remain same name as it is AntRangePicker and from  CustomRangePicker to RangePicker.